File: chapter-3:numerical-algorithms/gcd/python/euclidian-gcd.py
def gcd(a, b):
    while b != 0:
        remainder = a % b
        a = b
        b = remainder
    return a

print(gcd(25, 75))
print(gcd(78, 66))
print(gcd(1500, 32589))
print(gcd(56789, 12344567))

File: chapter-3_numerical-algorithms/gcd/javascript/euclidian-gcd.js
'use strict';
function gcd(a, b) {
  while(b !== 0) {
    var remainder = Math.abs(a % b);
    a = b;
    b = remainder;
  }
  return a;
}


console.log(gcd(75, 25));
console.log(gcd(78, 66));
console.log(gcd(1500, 32589));
console.log(gcd(56789, 12344567));
